scripts/ct/run.sh

```diff
@@ -130,6 +133,17 @@ for file in "${FILES[@]}"; do
     F_OPTIONS="$F_OPTIONS -f $file"
 done
 
+if [[ "${NEED_ROOT:-}" == 'yes' ]]; then
```

Comment:
Should we always set it to root when running in CI?

```suggestion
if [[ "${GITHUB_ACTIONS}" == true ]]; then
    NEED_ROOT=yes
fi

if [[ "${NEED_ROOT:-}" == 'yes' ]]; then
```